packages/db/src/migrations/0023_yeet_cognito.js

```diff
@@ -0,0 +1,19 @@
+import { sql } from "kysely"
```

Comment:
Maybe rename this to something more representative than just yeet cognito

Reply:
I do not care. It represent the action well enough